internal/cli/server_run.go

```diff
@@ -457,6 +521,31 @@ func (c *ServerRunCommand) Flags() *flag.Sets {
 			Usage:   acceptTOSHelp,
 			Default: false,
 		})
+		f.StringVar(&flag.StringVar{
+			Name:   "telemetry-opencensus-agent-addr",
+			Target: &c.flagTelemetryOpenCensusAgentAddr,
+			Usage: "Address of an opencensus agent or collector available to receive opencensus formatted\n" +
```

Comment:
```suggestion
			Usage: "Address of an OpenCensus agent or collector available to receive OpenCensus formatted " +
```

I removed the `\n`, we should let the CLI format new lines for long lines I think. /cc @evanphx